GetDKAN/dkan-tools
# Docs build command

Adds a command to build the docs.

Requires: GetDKAN/dkan-cli#5
Fixes #243 

## QA Steps

1. Build a DKAN site
2. Run `dktl dkan:docs`
3. Docs should build w/o error
